src/Extension.php

```diff
@@ -129,6 +129,36 @@ public static function init() {
 		add_filter( 'woocommerce_checkout_fields', [ __CLASS__, 'checkout_fields' ], 10, 1 );
 		add_action( 'woocommerce_checkout_update_order_meta', [ __CLASS__, 'checkout_update_order_meta' ], 10, 2 );
 
+		if ( \is_admin() ) {
+			\add_action(
+				'add_meta_boxes',
+				function( $post_type, $post ) {
+					if ( 'shop_order' !== $post_type ) {
+						return;
+					}
+
+					$order = \wc_get_order();
+
+					if ( ! $order instanceof WC_Order ) {
+						return;
+					}
+
+					\add_meta_box(
+						'woocommerce-order-pronamic-pay',
+						\__( 'Pronamic Pay', 'pronamic_ideal' ),
+						function( $post ) use ( $order ) {
+							include __DIR__ . '/../views/admin-meta-box-woocommerce-order.php';
+						},
+						$post_type,
+						'side',
+						'default'
+					);
+				},
```

Comment:
Move to an `Admin` class or at least a class method, instead of an anonymous function?

Reply:
I thought an admin controller class was a bit too much for just this. A separate function could be possible, but does that have any advantages here?

Reply:
IMHO, the `init()` function is getting a bit cluttered with an anonymous function. Calling `add_action()` with a clear function name improves readability of the code.